### DISCUSSION TITLE
Cannot activate student developer pack

### DISCUSSION THREAD

**Discussion by astraqiu** [community/community#54989] [GitHub Education]
### Hi everyone,

I've submitted a ticket several days ago #2142304, but have not heard from anyone yet. Posting here for more visibility.

Basically I've tried to activate the student developer pack but failed the verification step. I tried completing my profile with my full name, uploading my I-20 instead of my student ID card, and using campus VPN to visit the verification page, but none of these helped. Could anyone help me with this situation?

**Comment by ryanlua** [comment]
Hi @astraqiu, 

Sorry to hear that you are having issues with the Student Developer pack. You may have some common issues with your application which is listed below.

- **Check if your school is recognized by GitHub** Check to see if your school is in [GitHub Campus](https://education.github.com/schools) or [Microsoft Education Program](https://www.microsoft.com/en-in/education). If not, then you may need to contact GitHub Support for additional support.
- **Use up to date documents/information.** Provide the most up to date information and documents otherwise you might be rejected if it is too outdated.
- **Ensure your GitHub profile matches.** Make sure all information such as your name or location both match your profile and documents.
- **Make sure your school email is linked.** You should have your school email verified in your GitHub account.
- **Don't use a VPN.** GitHub may detect you using a VPN and automatically deny you access. Try applying without any VPN.
- **Translate your documents.** If your school is in another language than English than you should provide both the original untranslated document and English translated document.

You can read "[Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved)" and the "[Student Developer Pack Application & FAQs ➡️ Start Here #17814](https://github.com/orgs/community/discussions/17814)" discussion for more information about Global Campus rejections. 

As a reminder you do not need to have a Student ID to get the GitHub Student Developer Pack, for more information see "[How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)"

If none of this helps, you can contact GitHub for support at [Get help with GitHub Education](https://support.github.com/contact/education).

I hope this helps you with your question. Once resolved, you can mark this comment as the answer.

**Comment by samus-aran** [comment]
Hi @astraqiu 👋 

In order to qualify for the GitHub education program, we must be able to verify that you are either located close to your campus, or that you are a part of a verifiable distance learning program.

You will have the most success applying if you are able to do so from your school’s campus.

- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
- If you are still having difficulties enabling your location check out [this discussion on assessing your hardware or operating system's settings.](https://github.com/orgs/community/discussions/42246)
- If you are using a VPN, please disable it when you reapply. 
- If you are not using a VPN, try reapplying when connected to the internet from a different network.

I also suggest you ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

**Reply by muntahawani** [reply]
hey @samus-aran i have been try to apply while sitting at my campus but it still asks why am i so far from campus, even my location is enabled and website do have the permission to acess my location

**Comment by TaimoorCodes** [comment]
I hope your issue is resolved. Leaving this guide on [Medium](https://medium.com/@taimooor/how-to-get-your-github-student-developer-pack-5cab9f600da9) for anyone who are facing issues with getting their GitHub Student Developer Pack.
